src/Grpc/test/InteropTests/InteropTests.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using InteropTests.Helpers;
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace InteropTests
+{
+    public class InteropTests : IClassFixture<InteropTestsFixture>
+    {
+        private static readonly TimeSpan DefaultTimeout = TimeSpan.FromSeconds(30);
+
+        private readonly string _clientPath;
+        private readonly InteropTestsFixture _fixture;
+        private readonly ITestOutputHelper _output;
+
+        public InteropTests(InteropTestsFixture fixture, ITestOutputHelper output)
+        {
+            var attributes = Assembly.GetExecutingAssembly()
+                .GetCustomAttributes<AssemblyMetadataAttribute>();
+
```

Comment:
Same thing, can we have a fallback (or better yet a config file),  @javiercn can you drive coming up with a pattern that we can use in place of this assembly metadata stuff?  Components and Templates are the main consumers of this metadata pattern I think right now